docs/admin/kubelet.md

```diff
@@ -113,6 +113,7 @@ kubelet
       --network-plugin-dir="/usr/libexec/kubernetes/kubelet-plugins/net/exec/": <Warning: Alpha feature> The full path of the directory in which to search for network plugins
       --node-status-update-frequency=10s: Specifies how often kubelet posts node status to master. Note: be cautious when changing the constant, it must work with nodeMonitorGracePeriod in nodecontroller. Default: 10s
       --oom-score-adj=-999: The oom-score-adj value for kubelet process. Values must be within the range [-1000, 1000]
+      --outofdisk-transition-frequency=5m0s: Duration for which the kubelet has to wait before transitioning out of out-of-disk node condition status. Default: 300s
```

Comment:
I tried changing this before, but then I realized that this is an auto-generated file. It seems to parse 300s and automatically write it out as 5m0s. I have changed all the relevant occurrences to 5m0s for consistency.